scylla-artifacts.py

```diff
@@ -314,16 +317,30 @@ def run(self):
         if devlist:
             assert os.path.ismount('/var/lib/scylla'), "RAID setup failed, scylla directory isn't mounted rightly"
         # verify ntp
-        process.run('systemctl status ntpd')
+        if is_debian_variant:
+            process.run('service ntp status')
```

Comment:
There's a service manager library in avocado that handles abstracting these service related calls:

http://avocado-framework.readthedocs.io/en/52.0/api/utils/avocado.utils.html#module-avocado.utils.service

It contains checks such as the `is_systemd` function you implemented and more.